cfgov/ask_cfpb/models/django.py

```diff
@@ -556,14 +555,20 @@ def create_or_update_page(self, language=None):
             _parent = english_parent
             _slug = self.slug
             _question = self.question
-            _snippet = self.snippet
-            _answer = self.answer
+            # _snippet = self.snippet
```

Comment:
Do you want to remove these or leave them commented out?  In the next iteration we'll be removing this code entirely so it doesn't matter too much either way but might be easier to remove.